content/en/blog/_posts/2022-12-06-kubernetes-1.26-blog.md

```diff
@@ -0,0 +1,210 @@
+---
+layout: blog
+title: "Kubernetes v1.26: Electrifying"
+date: 2022-12-6
+slug: kubernetes-v1-26-release
+---
+
+**Authors**: [Kubernetes 1.26 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.26/release-team.md)
+
+It's with immense joy that we announce the release of Kubernetes v1.26!
+
+This release includes a total of 37 enhancements: eleven of them are graduating to Stable, ten are graduating to Beta, and sixteen of them are entering Alpha. We also have two features being deprecated or removed.
```

Comment:
@leonardpahlke adding comments here: we should decide on the approach here:

1. "12 features removed or deprectated", aligned with the list we have at the end of the article
2. "1 feature removed or deprecated", aligned with the tracked enhancements (but weird, because we then go on detailing 3 removals)
3. "12 features removed or deprecated, including 3 that we will detail below", mixing the total with the 3 that we have mentioned (CRI + 2 storage drivers)
3b. Same, but if removing something from the major themes, update the number.

Reply:
nr. 3